src/Sidekick.Core/Natives/NativeClipboard.cs

```diff
@@ -1,9 +1,54 @@
 using System.Threading.Tasks;
+using Sidekick.Core.Loggers;
+using Sidekick.Core.Settings;
 
 namespace Sidekick.Core.Natives
 {
     public class NativeClipboard : INativeClipboard
     {
+        private readonly SidekickSettings settings;
+        private readonly INativeKeyboard keyboard;
+        private readonly ILogger logger;
+
+        public NativeClipboard(SidekickSettings settings, INativeKeyboard keyboard, ILogger logger)
+        {
+            this.settings = settings;
+            this.keyboard = keyboard;
+            this.logger = logger;
+        }
+
+        public async Task<string> Copy()
+        {
+            var clipboardText = string.Empty;
+
+            if (settings.RetainClipboard)
+            {
+                clipboardText = await GetText();
+            }
+
+            await SetText(string.Empty);
+
+            keyboard.SendCommand(KeyboardCommandEnum.Copy);
+
+            await Task.Delay(100);
+
+            // Retrieve clipboard.
+            var itemText = await GetText();
```

Comment:
Kind of picky, but this class should be unaware of terms like "item"

Reply:
@leMicin You talking about the variable name or the fact I moved the copy function here? The variable should just be `text`

Reply:
I am talking about the variable name, I like the move of the function.

Reply:
@leMicin update in both places.